ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -93,10 +103,16 @@ object LedgerApiTestTool {
       else config.included
 
     val testsToRun = Tests.all.filterKeys(included -- config.excluded)
+    val performanceTestsToRun =
+      Tests.performanceTests(config.performanceTestsReportFile).filterKeys(config.performanceTests)
 
-    if (testsToRun.isEmpty) {
+    if (testsToRun.isEmpty && performanceTestsToRun.isEmpty) {
       println("No tests to run.")
       sys.exit(0)
+    } else if (config.included.nonEmpty && performanceTestsToRun.nonEmpty) {
+      println(
+        "Both regular tests and performance tests have been specified but only one type is allowed.")
+      sys.exit(-1)
```

Comment:
Why `-1`? Most shells will convert this to `255`, which is considered to be [in the class of "serious error"](https://www.gnu.org/software/libc/manual/html_node/Exit-Status.html).

Above, we use an exit status of `2` to signify bad configuration.

We use `64` elsewhere, which typically means "bad command line usage" [on BSD](https://man.openbsd.org/sysexits).

Don't mind which you pick, but < 128, please, and please change the other one to match. 😉